wagtailsharing/tests/test_models.py

```diff
@@ -84,3 +88,54 @@ def test_root_url_port_443_https(self):
     def test_root_url_other_port_http(self):
         site = SharingSite(hostname="test.hostname", port=1234)
         self.assertEqual(site.root_url, "http://test.hostname:1234")
+
+
+class TestShareableRoutablePage(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+        self.default_site = Site.objects.get(is_default_site=True)
+        self.sharing_site = SharingSite.objects.create(
+            site=self.default_site,
+            hostname="hostname",
```

Comment:
Minor nitpick, but it would make these tests a little easier to follow if this hostname were something more obviously for sharing, like "sharinghostname".

Reply:
Fixed in 78b0f45.